chall07/migferna.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func rgbToHexadecimal(response string) (int, int, int, int) {
+	str := strings.Split(response, " ")
+	data := str[0]
+	values := strings.Split(data, ",")
+	id, error := strconv.Atoi(strings.Split(values[0], "=")[1])
```

Comment:
It is wrong to call a variable `error` in Go. `error` is the [builtin error type interface](https://golang.org/pkg/builtin/#error), which means that from this point in your function you can't use this type anymore.  
Here it doesn't cause any problems, but it might in fact cause a lot of problem on error handling on more complicated projects.

It is common to call it `err`.